packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -19,15 +19,15 @@ import {FacetValueState} from '../../../features/facets/facet-api/value';
 import {SearchEngine} from '../../../app/search-engine/search-engine';
 import {
   buildCoreFacet,
-  CoreFacet,
-  CoreFacetState,
   Facet,
   FacetProps,
   FacetSearch,
   FacetSearchState,
   FacetState,
   FacetValue,
   SpecificFacetSearchResult,
+  CoreFacet,
+  CoreFacetState,
```

Comment:
Did you add these to not make a breakin change?

Reply:
yes